ports/esp32/Makefile

```diff
@@ -553,7 +661,14 @@ $(BUILD)/%.o: %.cpp
 BOOTLOADER_LIB_DIR = $(BUILD)/bootloader
 BOOTLOADER_LIB_ALL =
 
+ifeq ($(ESPIDF_CURHASH),$(ESPIDF_SUPHASH_V4))
+$(BUILD)/bootloader/$(ESPCOMP)/%.o: CFLAGS += -DBOOTLOADER_BUILD=1 -I$(ESPCOMP)/bootloader_support/include_priv -I$(ESPCOMP)/bootloader_support/include -I$(ESPCOMP)/micro-ecc/micro-ecc -I$(ESPCOMP)/efuse/include -I$(ESPCOMP)/esp_rom/include -Wno-error=format \
```

Comment:
Remove micro-ecc include from this line.

Reply:
Also, the `# libmicro-ecc.a` section below needs to be wrapped in `ifeq ($(ESPIDF_CURHASH),$(ESPIDF_SUPHASH_V3))`

Reply:
Also the `BOOTLOADER_LIBS += -L$(ESPCOMP)/esp32/lib -lrtc` line needs to be changed to the following for V4 (so conditional):
```
BOOTLOADER_LIBS += -L$(ESPCOMP)/esp_wifi/lib_esp32 -lrtc
```